web/packages/teleport/src/lib/useMfa.ts

```diff
@@ -41,11 +41,64 @@ export function useMfa(emitterSender: EventEmitterMfaSender): MfaState {
     totpChallenge: false,
   });
 
-  // TODO (avatus), this is stubbed for types but will not be called
-  // until SSO as MFA backend is in.
+  function clearChallenges() {
+    setState(prevState => ({
+      ...prevState,
+      totpChallenge: false,
+      webauthnPublicKey: null,
+      ssoChallenge: null,
+    }));
+  }
+
+  // open a broadcast channel if sso challenge exists so it can listen
+  // for a confirmation response token
```

Comment:
I'm gonna need more details on how it's going to work. Do you have an RFD? I'm interested in who talks to that broadcast channel and where the `ssoChallenge` (and `channelId`) come from.

Reply:
https://github.com/gravitational/teleport/pull/44699

The window pop up we use during the SSO ceremony is what creates and broadcasts the token from a broadcast channel from its confirmation page. Check out the e PR linked in this PRs description